test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -69,5 +69,14 @@ Describe "Start-Process" -Tags @("CI","SLOW") {
 	$dirEntry.Length | Should BeGreaterThan 0
     }
 
+	It "Should give an error when Verb parameter is used" {
```

Comment:
The error shouldn't happen in Windows PowerShell - right? So I'd expect a `-Skip`.